CMakeLists.txt

```diff
@@ -22,15 +24,33 @@ set(tinyobjloader-examples-objsticher
 	${TINYOBJLOADEREXAMPLES_DIR}/obj_sticher/obj_sticher.cc
 	)
 
+#Install destinations
+set(TINYOBJLOADER_CMAKE_DIR lib/cmake)
```

Comment:
...this is why `LIB_SUFFIX` is nice... anyone that knows about `LIB_SUFFIX` knows that the install location currently is wrong. Anyone that doesn't know or care isn't setting `LIB_SUFFIX`, and won't see a change.